packages/ckeditor5-dev-release-tools/lib/release/updateversions.js

```diff
@@ -0,0 +1,80 @@
+/**
+ * @license Copyright (c) 2003-2023, CKSource Holding sp. z o.o. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+'use strict';
+
+const { globSync } = require( 'glob' );
+const fs = require( 'fs-extra' );
+const semver = require( 'semver' );
+const { normalizeTrim, join, toUnix } = require( 'upath' );
+const { tools } = require( '@ckeditor/ckeditor5-dev-utils' );
+const getPackageJson = require( '../utils/getpackagejson' );
+
+/**
+ * Updates version of root package and subpackages if `packagesDirectory` path is provided.
+ *
+ * @param {Object} options
+ * @param {String} options.version
+ * @param {String} [options.packagesDirectory]
+ * @param {String} [options.cwd]
+ */
+module.exports = function updateVersions( { packagesDirectory, version, cwd = process.cwd() } ) {
+	const normCwd = toUnix( cwd );
+	const normPackagesDir = packagesDirectory ? normalizeTrim( packagesDirectory ) : packagesDirectory;
+	const packagesGlobs = normPackagesDir ? [ normPackagesDir + '/*', './' ] : './';
+	const packagesPaths = globSync( packagesGlobs, { cwd: normCwd, absolute: true } );
+
+	checkVersionGreaterThanOldVersion( version, getPackageJson( normCwd ).version );
+	checkVersionIsAvailableInNpm( version, getRandomPackageJson( normPackagesDir, normCwd ).name );
+
+	for ( const packagePath of packagesPaths ) {
+		const packageJsonPath = join( packagePath, 'package.json' );
```

Comment:
Because `glob` returns directory paths instead of paths to `package.json` files, I guess we should make sure that `package.json` file actually exists in each found directory. Sometimes there are empty directories left between git checkouts and in that case this script will crash during the release.